docs/docs/userguide/monitoring/PROMETHEUS.md

```diff
@@ -0,0 +1,142 @@
+# Monitoring with Prometheus
+
+The instructions outlined on this page provide details on how you can enable [Prometheus](https://prometheus.io/){.external} monitoring on your stack with [Grafana](https://grafana.com/){.external}
+
+## 1. Install Prometheus stack
+
+!!!tip "Note"
+    * This approach will also install Grafana
+    * For the purposes of this guide the Prometheus stack will be installed with the release name `prometheus-stack`.
+
+Install [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack){.external} Helm chart
+
+Fetch the repo and perform an update
+```bash
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo update
+```
+Now install the Prometheus stack
+```bash
+helm install prometheus-stack prometheus-community/kube-prometheus-stack
+```
+
+## 2. Expose JMX metrics
+
+Follow [these instructions](../../../userguide/OPERATION/#expose-jmx-metrics) for details on how to enable and expose `JMX` for your product via a dedicated `Service`. 
+
+
+## 3. Create a ServiceMonitor
```

Comment:
I wonder if we share a link to prometheus operator docs which explain what service monitor is.

Reply:
Good call Ill do that.